recipes/dipy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.11.0" %}
+
+package:
+  name: dipy
+  version: {{ version }}
+
+source:
+  fn: dipy-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
+  md5: 7723e8994d31e910d47b7a0cf064a3d8
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
I suspect that, since this requires numpy and cython, it uses the numpy ABI. If That is true you need to use: `numpy x.x` in the `build` and `run` fields.